cleanrl/td3_continuous_action_jax.py

```diff
@@ -248,7 +250,7 @@ def actor_loss(params):
         qf2_state = qf2_state.replace(
             target_params=optax.incremental_update(qf2_state.params, qf2_state.target_params, args.tau)
         )
-        return actor_state, qf1_state, actor_loss_value
+        return actor_state, qf1_state, actor_loss_value, grad_norm
```

Comment:
I suppose the `grad_norm` is for debugging purposes? Please remove it in the final version.